docs/lib/graphqlapi/fragments/flutter/advanced-workflows/20_custom.md

```diff
@@ -0,0 +1,27 @@
+```dart
+try{
+    String graphQLDocument = '''query getTodo(id: \$todoId) {
+      getTodo(id: \$todoId) {
+        id
+        name
+        description
+      }
+    }''';
+
+    var operation = await Amplify.API.query<String>(
```

Comment:
Same, we probably don't need await here.

Reply:
updated